app/lib/allowed_cross_namespace_usage.rb

```diff
@@ -8,4 +8,5 @@ module AllowedCrossNamespaceUsage
   VendorAPIApplicationPresenter = VendorAPI::ApplicationPresenter
   RegisterAPISingleApplicationPresenter = RegisterAPI::SingleApplicationPresenter
   VENDOR_API_VERSION = VendorAPI::VERSION
+  ProviderInterfaceNewNoteForm = ProviderInterface::NewNoteForm
```

Comment:
instead of porting over the notes form, we have been extracting the validation of both, and reusing everywhere (see the offer validation as an example) is it worth doing the same here?

Reply:
I've removed this completely and refactored the form to use a sub-service which is shared. https://github.com/DFE-Digital/apply-for-teacher-training/pull/6338/commits/3276485527598c28d0a79cc16d739f48d763ae12